stable/goldilocks/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 appVersion: "v3.1.4"
-version: 3.2.7
+version: 3.2.8
```

Comment:
Because of the possibility that someone would have breaking changes here if they are deploying currently with an ingress path definition that does not include `type` I think we should make this a major release version change to `4.0.0`